### ISSUE TITLE
Missing events?

### ISSUE THREAD

**Issue by madhikarma** [amplitude/Amplitude-iOS#334]
## Summary

Hi there

We were looking at a situation (10% of the time) where events are being received and shown by our internal Analytics system but missing in Amplitude for the same user.

Are there situations where the events are not going to be sent by the SDK i.e. dropped from being sent or ignored by the Amplitude servers?

In the particular case the events missing occur after opening from a push notification and before the app going into the background mode.

Thank you

**Comment by haoliu-amp** [comment]
Hello, in order to help you further, pls file a ticket here https://help.amplitude.com/hc/en-us/requests/new. Thanks!

**Comment by ZkHaider** [comment]
@haoliu-amp closing this issue doesn’t technically answer portions of this question, for example it might be helpful to state how Amplitude flushes events internally and so on, before asking a user to submit a new ticket request.

We all are trying to help each other and delegating an issue off to a detached process (submit a ticket) comes across as dismissive.

**Comment by madhikarma** [comment]
Thanks @ZkHaider for the support 

fwiw: I emailed / created a ticket as suggested and got an email with technical support confirming we have errors from Amplitude servers (possibly too many events) for our app but I haven't followed up on that yet

But yes I was hoping for some technical insight here about the Amplitude SDK side but I must admit it did feel a little dismissive at the time to get [this response](https://github.com/amplitude/Amplitude-iOS/issues/334#issuecomment-810429939) in a community as such as GitHub (or at least give a reason why a ticket was preferred route for this type of query)

I was expecting to get some info or links to the architecture or common areas when replying to people in an OSS context

Will share if we have any findings here once our investigation is done that might be of use to others in the community

**Comment by haoliu-amp** [comment]
@madhikarma Sorry to hear that this made you felt that way. It might be tricky if someone claim that events are missing and we might need to request instrumentation code from you. To better protect your privacy, that's why we routed you to our support. 

We're revamping our dev doc these days, and we'll consider to add those info you talked about at the same time.

cc: @ZkHaider
